packages/rainbowkit/src/components/ConnectButton/formatENS.ts

```diff
@@ -1,4 +1,8 @@
-export function formatENS(name: string): string {
+import { GetEnsNameReturnType } from 'viem';
+
+export function formatENS(name: GetEnsNameReturnType): string {
+  if (!name) return '';
+
```

Comment:
This won't break since a lot of places in code we make sure it's true before formatting it. Just an empty string with early return works fine for now.

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/c3aa75c9-9dd9-43ec-ae10-a193bb3c0db1)